src/styles/utilities.less

```diff
@@ -164,3 +164,11 @@
 .u-overflow-auto {
   overflow: auto;
 }
+
+.u-readonly-input {
+  color: @edu-text-color-secondary !important;
+
+  & > span {
+    color: @edu-text-color-secondary !important;
+  }
```

Comment:
We also have to add

~~~
  & > span {
    color: @edu-text-color-secondary !important;
  }
~~~

in case of multi-line inputs (or go with `& > *` to cover all children?)